packages/ui-kit/src/Timeline/implementation/vis-timeline/toVisTimelineOptions.tsx

```diff
@@ -41,7 +46,11 @@ export const toVisTimelineOptions = (options: TimelineOptions): VisTimelineOptio
       template: (originalItemData, parsedItemData) => {
         const wrapper = document.createElement('div');
         const root = createRoot(wrapper);
-        root.render(<TooltipComponent {...originalItemData} />);
+
+        flushSync(() => {
+          root.render(<TooltipComponent {...originalItemData} />);
+        });
```

Comment:
I think we need to remove the template option, since we can't get it work

Reply:
@Akiyamka Remove TooltipComponent template?

Reply:
remove ability to define custom component in out <Timeline /> component

Reply:
Done